roles/beaconchain_explorer_aio/tasks/setup.yml

```diff
@@ -62,6 +62,14 @@
     group: "{{ beaconchain_explorer_aio_user }}"
     mode: '0640'
 
+- name: Create frontend config file
+  ansible.builtin.copy:
+    content: "{{ beaconchain_frontend_config }}"
```

Comment:
The variable `beaconchain_frontend_config` is undefined. Missing in defaults/main.yml ?

Reply:
Also, if the var should exist in defaults, it should be correctly prefixed like the other vars.

Reply:
updated with defaults